apps/web/pages/settings/admin/lockedSMS/UsersTable.tsx

```diff
@@ -0,0 +1,147 @@
+  users?: User[];
+  teams?: Team[];
+  setSMSLockState: (param: { userId?: number; teamId?: number; lock: boolean }) => void;
+}) => {
+  function getActions({ user, team }: { user?: User; team?: Team }) {
+    const smsLockState = user?.smsLockState ?? team?.smsLockState;
+    if (!smsLockState) return [];
+
+    const actions = [
+      {
+        id: "unlock-sms",
+        label: smsLockState === SMSLockState.LOCKED ? "Unlock SMS sending" : "Lock SMS sending",
+        onClick: () =>
+          setSMSLockState({
+            userId: user ? user.id : undefined,
+            teamId: team ? team.id : undefined,
+            lock: smsLockState !== SMSLockState.LOCKED,
+          }),
+        icon: "lock" as IconName,
+      },
+    ];
+    if (smsLockState === SMSLockState.REVIEW_NEEDED) {
+      actions.push({
+        id: "reviewed",
+        label: "Mark as Reviewed",
+        onClick: () =>
+          setSMSLockState({
+            userId: user ? user.id : undefined,
+            teamId: team ? team.id : undefined,
+            lock: false,
+          }),
+        icon: "pencil" as IconName,
+      });
+    }
+
+    return actions;
+  }
+
+  return (
+    <>
+      <Table>
+        <Header>
+          <ColumnTitle widthClassNames="w-auto">User/Team</ColumnTitle>
+          <ColumnTitle>Status</ColumnTitle>
+          <ColumnTitle widthClassNames="w-auto">
+            <span className="sr-only">Edit</span>
+          </ColumnTitle>
+        </Header>
+
+        <tbody className="divide-subtle divide-y rounded-md">
+          {users.map((user) => (
+            <Row key={`user-${user.id}`}>
+              <Cell widthClassNames="w-auto">
+                <div className="min-h-10 flex items-center">
+                  {" "}
+                  <Avatar
+                    size="md"
+                    alt={`Avatar of ${user.username || "Nameless"}`}
+                    imageSrc={`${WEBAPP_URL}/${user.username ? user.username : "default"}/avatar.png`}
+                  />
+                  <div className="text-subtle ml-4 font-medium">
+                    <span className="text-default">{user.name}</span>
+                    <span className="ml-3">/{user.username}</span>
+                    <br />
+                    <span className="break-all">{user.email}</span>
+                  </div>
+                </div>
+              </Cell>
+
+              <Cell>{user.smsLockState}</Cell>
+              <Cell widthClassNames="w-auto">
+                <DropdownActions actions={getActions({ user })} />
+              </Cell>
+            </Row>
+          ))}
+          {teams.map((team) => (
+            <Row key={`team-${team.id}`}>
+              <Cell widthClassNames="w-auto">
+                <div className="min-h-10 flex items-center">
+                  <Avatar
+                    size="md"
+                    alt={`Avatar of ${team.name}`}
+                    imageSrc={`${WEBAPP_URL}/${team.slug ? team.slug : "default"}/avatar.png`}
```

Comment:
here we should probably use  const profileImageSrc = getPlaceholderAvatar(team.logoUrl || team.parent?.logoUrl, team.name);